### ISSUE TITLE
Inclure les mappings importés dans l'onglet "Mappings" d'une ontologie

### ISSUE THREAD

**Issue by saubin78** [agroportal/agroportal_web_ui#65]
Mon ontologie contient nativement des mappings skos. Ces mappings apparaissent bien sur la page des concepts concernés de mon ontologie mais pas dans l'onglet "Mappings" (sans doute réservé aux mappings calculés par AgroPortal).
Proposition :  inclure les mappings déclarés dans mon ontologies à l'onglet "Mappings" avec un statut particulier peut-être.

Sujet (pas super sexy) pour un hackathon?

**Comment by jonquet** [comment]
Oui. Cela fait des années que je le mentionne. Les mappings ne sont pas parsés et extraits. Je vais réactiver cela chez eux.

De notre coté, nous le faisons a la main. @vemonet a des scripts pour:
- extraire des mappings des format OWL ou SKOS
- les representer au format JSON que Bio/AgroPortal prend
- les charger automatiquement dans AgroPortal

@vemonet tu confirmes ?

**Comment by jonquet** [comment]
Hi @mdorf , @jvendetti 
This topic is about exporting automatically class mapping when they are explicitly defined in the ontology file. 
Mappings can be defined with multiple properties (e.g., owl:sameAs, skos:exactMatch, etc) but usually the syntax is quite common. 
Without such a feature, users have to:
- upload their ontologies to BP (source and target)
- extract their mappings (maybe stored in one of the ontologies) and represent them in JSON
- push them via automatic script to BioPortal
As a result most of them never do it and the mapping repository does not fill in. 

Before looking into this, you might be interested in pulling our code changes to store "external" mappings: https://github.com/agroportal/documentation/wiki/Mappings that @vemonet can point to.
